src/defaultsettings.cpp

```diff
@@ -26,6 +26,68 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "mapgen/mapgen.h" // Mapgen::setDefaultSettings
 #include "util/string.h"
 
+
+
+bool detect_touch()
+{
+#if ENABLE_TOUCH  // Android & special builds
```

Comment:
Do we still need ENABLE_TOUCH if we have auto-detection? We could check for Android directly.

Reply:
We could. Think to account for is eventually people with Android using keyboard + mouse, but it's just a default so I wouldn't mind to hardcode `ANDROID`

Reply:
Well, it's hardcoded anyway.